### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by fao89**
Adding how to file an issue.

[noissue]

### CODE REVIEW

**Review by mdellweg — APPROVED** [review]
Thank you!
Is this from the plugin_template?

File: README.rst [hunk 1]
```diff
@@ -11,3 +11,45 @@ images, similar to the pulp_docker plugin for Pulp 2.
...
+     - Strive to be specific and concise.
+
+   * - Description
+     - This is the most important part! Please see `Description Field <https://docs.pulpproject.org/pulpcore/bugs-features.html#issue-description>`_.
+
+   * - Category
+     - Choose one if applicable, blank is OK.
+
+   * - Version
+     - The version of pulpcore that you discovered the issue.
```
Comment (mdellweg) [inline README.rst:48]: Shouldn't this be the pulp_container version?

**Comment by fao89** [comment]
> Thank you!
> Is this from the plugin_template?

not yet

File: README.rst (hunk 1 above)
Comment (fao89) [reply README.rst:48]: thank you, it was a bad copy/paste

**Comment by ipanova** [comment]
@fao89 can you please squash the commits?

**Pull request merged**